products.php

```diff
@@ -10,22 +10,22 @@
 
 try
 {
-    $jsonString = [];
+    $jsonArray = [];
```

Comment:
No need to do this if you're passing the empty array into the error response below (as described above)